recipe/meta.yaml

```diff
@@ -1,47 +1,44 @@
 {% set name = "spyder-kernels" %}
 {% set version = "2.4.0" %}
-{% set hash = "3bd32fbe30a150ab9ccedbd5beb6ddd38bde1d9a8778da52f7edf420bbfb8e51" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ hash }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/spyder-kernels-{{ version }}.tar.gz
+  sha256: 3bd32fbe30a150ab9ccedbd5beb6ddd38bde1d9a8778da52f7edf420bbfb8e51
 
 build:
-  number: 0
-  script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 1
+  noarch: python
+  string: "unix_pyh{{ PKG_HASH }}_{{ PKG_BUILDNUM }}"  # [unix]
+  string: "win_pyh{{ PKG_HASH }}_{{ PKG_BUILDNUM }}"   # [win]
```

Comment:
What's this needed for? I haven't seen it before in conda recipes.

Reply:
I think it came from one of the iterations of a new PR to document how to make a package `noarch: python` conda-forge/conda-forge.github.io#1839 but from the discussion there is not needed.

Reply:
I think it's still needed, only when https://github.com/conda/conda-build/pull/4606 is released should it be fine.